clippy_lints/src/utils/mod.rs

```diff
@@ -566,7 +566,7 @@ impl<'a> DiagnosticWrapper<'a> {
     fn wiki_link(&mut self, lint: &'static Lint) {
         if env::var("CLIPPY_DISABLE_WIKI_LINKS").is_err() {
             self.0.help(&format!(
-                "for further information visit https://github.com/rust-lang-nursery/rust-clippy/wiki#{}",
+                "for further information visit https://rust-lang-nursery.github.io/rust-clippy/master/index.html#{}",
```

Comment:
This should not point to master but the current package version `CARGO_PKG_VERSION`.

Reply:
Github masked this comment but it is still valid.